intera_examples/scripts/go_to_cartesian_pose.py

```diff
@@ -86,6 +86,9 @@ def main():
     parser.add_argument(
         "--rotational_accel", type=float, default=1.57,
         help="The max rotational acceleration of the endpoint (rad/s/s)")
+    parser.add_argument(
+        "--timeout", type=float, default=None,
```

Comment:
There are definitely trajectories which take longer than 30 seconds if they are moving slow. I would default at least longer than 60 seconds, maybe 120 seconds.

In general though, I think that users are always surprised when they hit unknown timeout limits.

Reply:
I would agree in general, however, I want to prevent against an estop situation where the robot is still communicating but cannot move, then once it is re-enabled, trying to pick up the trajectory again.

Reply:
My vote would be to keep the default (no timeout), since that behavior has been working well for now, and there are foreseeable use cases with very long trajectories that are valid. My understanding was that this timeout was primarily used in situations where the robot shut down.

Question: @asthakukreja - what happens when the timeout is not set and the E-stop is triggered? What happens when it is re-enabled?

Reply:
When you hit E-stop, the timeout functionality is not kicked in. We get a false result. So we do not need to worry about timeout duration when E-stop is triggered. It is only during when the robot is shutdown or say when you disconnect internet, you could see the host machine still trying to communicate.

Reply:
I have submitted a new commit for review and have kept the default behavior to be no timeout. I think this will run the tests in the same way as originally designed.